examples/tutorials/rest-api-annotated-service/src/main/java/example/armeria/server/blog/BlogPostRequestConverter.java

```diff
@@ -0,0 +1,47 @@
+package example.armeria.server.blog;
+
+import java.lang.reflect.ParameterizedType;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.annotation.Nullable;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.armeria.common.AggregatedHttpRequest;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.annotation.RequestConverterFunction;
+
+/**
+ * Converts an {@link HttpRequest} to a {@link BlogPost}.
+ */
+final class BlogPostRequestConverter implements RequestConverterFunction {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private final AtomicInteger idGenerator = new AtomicInteger();
+
+    @Override
+    public Object convertRequest(ServiceRequestContext ctx, AggregatedHttpRequest request,
+                                 Class<?> expectedResultType,
+                                 @Nullable ParameterizedType expectedParameterizedResultType)
+            throws Exception {
+        if (expectedResultType == BlogPost.class) {
```

Comment:
How about adding a dedicated class for blog post creation request, so we don't need this converter at all?

Reply:
Let's add another section to the tutorial that we can do this in a simple way as we discussed. 😄